src/wallet/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ target_link_libraries(wallet
     common
     cryptonote_core
     mnemonics
+    device_trezor
```

Comment:
This should really link to device, not device_trezor. Is this a problem ?

Reply:
I think `device` gets linked to the `wallet` transitively via `cryptonote_core`.

The reason why I add `device_trezor` here is the `hw::device& wallet2::lookup_device(const std::string & device_descriptor)` for dynamic device registration and lookup to break cyclic dependencies - in the comment below.